File: fragmentFactory/app/src/main/java/com/example/fragmentfactory/MainActivity.java
package com.example.fragmentfactory;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        BlankFragment fragment =  BlankFragment.newInstance("example", 4);

        getSupportFragmentManager().beginTransaction().replace(R.id.container, fragment).commit();
    }
}

File: WidgetPracticeSkeleton/app/src/main/java/com/bitamirshafiee/widgetpracticeskeleton/MainActivity.kt
package com.bitamirshafiee.widgetpracticeskeleton

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.SeekBar
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        age_text_view.text = String.format(resources.getString(R.string.str_age), 0)
        seek_bar.setOnSeekBarChangeListener(
            object : SeekBar.OnSeekBarChangeListener {
                override fun onProgressChanged(p0: SeekBar?, p1: Int, p2: Boolean) {
                    age_text_view.text = String.format(resources.getString(R.string.str_age), p1)
                }

                override fun onStartTrackingTouch(p0: SeekBar?) {

                }

                override fun onStopTrackingTouch(p0: SeekBar?) {

                }

            }
        )
    }
}